be/src/util/compression/block_compression.cpp

```diff
@@ -44,6 +44,7 @@
 
 #include "gutil/endian.h"
 #include "gutil/strings/substitute.h"
+#include "thirdparty/libdeflate/libdeflate.h"
```

Comment:
Normally We don't include header file in this way.

We do like `#include <libdeflate/libdeflate.h>`

Reply:
suggest to  use contrib instead of thirdparty to place submodules